package.json

```diff
@@ -16,7 +16,7 @@
     "jasmine": "^2.3.0",
     "phantomjs": "^1.9.11",
     "sinon": "^1.10.3",
-    "uglify-js": "2.3.6",
+    "uglify-js": "2.6.3",
```

Comment:
If there's no diff before and after the upgrade I'd say it should be good to go.